src/librustc_mir_build/hair/pattern/check_match.rs

```diff
@@ -410,7 +410,7 @@ fn check_not_useful<'p, 'tcx>(
 ) -> Result<(), Vec<super::Pat<'tcx>>> {
     let wild_pattern = cx.pattern_arena.alloc(super::Pat::wildcard_from_ty(ty));
     let v = PatStack::from_pattern(wild_pattern);
-    match is_useful(cx, matrix, &v, ConstructWitness, hir_id, true) {
```

Comment:
Can you leave a note re. the `false` in this case?

Reply:
Added in 4b1aa9fa